### PR TITLE
fix(edgeless): create popper on null element

### LINKED ISSUE #7067: createButtonPopper on a null element
Sentry Issue: [AFFINE-5RW](https://toeverything.sentry.io/issues/5353633774/?referrer=github_integration)

```
TypeError: Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'.
  at createButtonPopper (webpack://@affine/web/../../../node_modules/@blocksuite/blocks/src/_common/utils/button-popper.ts:58:34)
  at EdgelessChangeNoteButton.updated (webpack://@affine/web/../../../node_modules/@blocksuite/blocks/src/root-block/widgets/element-toolbar/change-note-button.ts:400:33)
  at b._$AE (webpack://@affine/web/../../../node_modules/lit/node_modules/@lit/reactive-element/reactive-element.js:6:5094)
  at b.performUpdate (webpack://@affine/web/../../../node_modules/lit/node_modules/@lit/reactive-element/reactive-element.js:6:4956)
  at b.scheduleUpdate (webpack://@affine/web/../../../node_modules/lit/node_modules/@lit/reactive-element/reactive-element.js:6:4427)
...
(1 additional frame(s) were not displayed)
```

### PR DESCRIPTION
**Pull request by L-Sun**
Close #7067

### CODE REVIEW

**Comment by L-Sun** [comment]
* **#7066** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7066?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @L-Sun and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/blocks/src/root-block/widgets/element-toolbar/change-note-button.ts [hunk 1]
```diff
@@ -226,7 +226,7 @@ export class EdgelessChangeNoteButton extends WithDisposable(LitElement) {
   @query('.display-mode-button-group')
   private _displayModeButtonGroup!: HTMLDivElement;
   @query('note-display-mode-panel')
-  private _displayModePanel!: HTMLDivElement;
+  private _displayModePanel!: HTMLDivElement | null;
```
Comment (fourdim) [inline packages/blocks/src/root-block/widgets/element-toolbar/change-note-button.ts:229]:
```ts
private _displayModePanel: HTMLDivElement | undefined;
```
Remove that non-null assertion operator (postfix !)

Comment (L-Sun) [reply packages/blocks/src/root-block/widgets/element-toolbar/change-note-button.ts:229]: I think the `@query` always returns an `Element` or `null` according to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector#return_value). If I use `HTMLDivElement | null` and remove the postfix !, ESLint will raise a "no initializer" error (ts 2564)

**Review by doodlewind — APPROVED** [review]
LGTM

**Pull request merged**